tests/test_git.py

```diff
@@ -945,3 +988,64 @@ def test_fetch(
 
     target.fetch()
     assert target.get_ref("refs/remotes/origin/master") == rev
+
+
+@pytest.mark.skip_git_backend("pygit2", "gitpython")
+@pytest.mark.parametrize("untracked_files", ["all", "no"])
```

Comment:
I felt a bit conflicted when this test, because I feel like this is testing something that is already tested in `dulwich`, but this is basically testing that the `untracked_files` and `ignored` is correctly passed down to `dulwich`'s `status`

Reply:
I'm not sure I follow?

Testing this here is correct because in scmrepo we should be testing that scmrepo's returned `status()` is the same regardless of our git backend.